### PR TITLE
Wc2-106: small adjustments to the get storage blacklisted endpoint

### PR DESCRIPTION
**Pull request by niconoe**
The response to GET /api/mobile/storage/blacklisted had a few differences to what was specified in https://bluesquare.atlassian.net/browse/WC2-65, in particular:

.- `updated_at` and `created_at` fields were returned at the root (device) and shouldn’t be.
- the `status` dict/object should be renamed `storage_status`
- `storage_status.updated_at` returned an ISO8601 string rather than a timestamp
- details about the orgunit and entity linked to the storage should not be returned

## Self proof reading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [x] Are there enough tests

## How to test

- Make sure you have at least one device in the database with the `BLACKLISTED` status
- Go to `/api/mobile/storage/blacklisted/`
- Make sure the response conforms to what is described at https://bluesquare.atlassian.net/browse/WC2-65
- (after the mobile app has been updated): make sure the interactions to detect the blacklisted devices still work

Explain how to test your PR.
If a specific config is required explain it here (dataset, account, profile, ...)

@bmonjoie: I think the backend now complies fully with the specs, can you do the adjustments to the mobile app and make sure all works as expected?

### CODE REVIEW

File: iaso/tests/api/test_storage.py [hunk 1]
```diff
@@ -434,7 +435,12 @@ def test_list_base(self):
                     "created_at": 1580608922.0,
                     "storage_id": "ANOTHER_EXISTING_STORAGE_BLACKLISTED_STOLEN",
                     "storage_type": "NFC",
-                    "status": {"status": "BLACKLISTED", "reason": "STOLEN", "comment": "", "updated_at": None},
+                    "status": {
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: This should be `storage_status`

File: iaso/tests/api/test_storage.py [hunk 2]
```diff
@@ -463,7 +469,12 @@ def test_list_filter_by_status(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: This should be `storage_status`

File: iaso/tests/api/test_storage.py [hunk 3]
```diff
@@ -494,7 +505,12 @@ def test_list_filter_by_reason(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: This should be `storage_status`

File: iaso/tests/api/test_storage.py [hunk 4]
```diff
@@ -523,7 +539,12 @@ def test_list_filter_by_type(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: This should be `storage_status`

File: iaso/tests/api/test_storage.py [hunk 5]
```diff
@@ -1258,22 +1279,24 @@ def test_get_blacklisted_devices(self):
...
-                        "created_at": 1580608922.0,
                         "storage_id": "ANOTHER_EXISTING_STORAGE_BLACKLISTED_ABUSE",
                         "storage_type": "SD",
-                        "status": {"status": "BLACKLISTED", "reason": "ABUSE", "comment": "", "updated_at": None},
-                        "org_unit": None,
-                        "entity": {"id": self.entity.id, "name": "New Client 3"},
+                        "storage_status": {
+                            "status": "BLACKLISTED",
+                            "reason": "ABUSE",
+                            "updated_at": None,
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: `None` is not an acceptable response.

**Review by bmonjoie — CHANGES_REQUESTED** [review]

File: iaso/tests/api/test_storage.py (hunk 1 above)
Comment (SimonDus) [reply iaso/tests/api/test_storage.py]: Indeed

**Comment by niconoe** [comment]
Remaining changes noted by @bmonjoie were implemented. @bmonjoie / @beygorghor : can you make sure the mobile and frontend are also ajusted to deal with those last changes (status -> storage_status for both the storage list and and the "logs per device" endpoints)?

File: iaso/tests/api/test_storage.py [hunk 6]
```diff
@@ -520,7 +547,7 @@ def test_list_filter_by_type(self):
                     "created_at": 1580608922.0,
                     "storage_id": "EXISTING_STORAGE",
                     "storage_type": "NFC",
-                    "status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
+                    "storage_status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 7]
```diff
@@ -423,15 +425,15 @@ def test_list_base(self):
...
+        self.assertListEqual(
             received_json,
             [
                 {
                     "updated_at": 1580608922.0,
                     "created_at": 1580608922.0,
                     "storage_id": "EXISTING_STORAGE",
                     "storage_type": "NFC",
-                    "status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
+                    "storage_status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 8]
```diff
@@ -729,7 +766,7 @@ def test_get_logs_for_device_base(self):
                 "created_at": 1580608922.0,
                 "storage_id": "EXISTING_STORAGE",
                 "storage_type": "NFC",
-                "status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
+                "storage_status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 9]
```diff
@@ -766,7 +803,7 @@ def test_get_logs_for_device_ou_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 10]
```diff
@@ -797,7 +834,7 @@ def test_get_logs_for_device_ou_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 11]
```diff
@@ -834,7 +871,7 @@ def test_get_logs_for_device_types_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 12]
```diff
@@ -867,7 +904,7 @@ def test_get_logs_for_device_types_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 13]
```diff
@@ -885,7 +922,7 @@ def test_get_logs_for_device_types_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 14]
```diff
@@ -927,7 +964,7 @@ def test_get_logs_for_device_types_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 15]
```diff
@@ -978,7 +1015,7 @@ def test_get_logs_for_device_status_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 16]
```diff
@@ -1011,7 +1048,7 @@ def test_get_logs_for_device_status_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 17]
```diff
@@ -1032,7 +1069,7 @@ def test_get_logs_for_device_reason_filter(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 18]
```diff
@@ -1085,7 +1122,7 @@ def test_get_logs_for_device_pagination(self):
                     "created_at": 1580608922.0,
                     "storage_id": "EXISTING_STORAGE",
                     "storage_type": "NFC",
-                    "status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
+                    "storage_status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 19]
```diff
@@ -1125,7 +1162,7 @@ def test_get_logs_for_device_pagination(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

File: iaso/tests/api/test_storage.py [hunk 20]
```diff
@@ -1165,7 +1202,7 @@ def test_get_logs_for_device_pagination(self):
...
```
Comment (bmonjoie) [inline iaso/tests/api/test_storage.py]: I believe that when status is "OK", the reason and comment should be null. Updated at should be set

**Review by bmonjoie — CHANGES_REQUESTED** [review]

**Pull request merged**